src/modding/ModManager.cs

```diff
@@ -558,7 +850,35 @@ private void UpdateSelectedModInfo()
 
             leftArrow.Disabled = true;
             rightArrow.Disabled = true;
+            moveModUpButton.Disabled = true;
+            moveModDownButton.Disabled = true;
+        }
+
+        errorInfoLabel.Hide();
+    }
+
+    private List<ImageTexture> LoadModPreviewImages(FullModDetails mod)
+    {
+        var returnValue = new List<ImageTexture>();
+
+        if (mod.Info.PreviewImages != null)
+        {
+            foreach (string currentImagePath in mod.Info.PreviewImages)
+            {
+                if (string.IsNullOrEmpty(currentImagePath))
+                    return null!;
```

Comment:
Why is this suddenly returning null here? And even overwriting the nullability. This looks super wrong.